### PR TITLE
zfsbackup: update unstable-2019-03-05 -> unstable-2020-09-30

### PR DESCRIPTION
**Pull request by rajivr**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/backup/zfsbackup/deps.nix [hunk 1]
```diff
@@ -1,273 +1,1326 @@
-# This file was generated by https://github.com/kamilchm/go2nix v1.3.0
+# file generated from go.mod using vgo2nix (https://github.com/adisbladis/vgo2nix)
```
Comment (SuperSandro2000) [inline pkgs/tools/backup/zfsbackup/deps.nix:1]: You should use https://github.com/nix-community/vgo2nix or buildGoModule.

File: pkgs/tools/backup/zfsbackup/default.nix [hunk 2]
```diff
@@ -2,16 +2,16 @@
 
 buildGoPackage rec {
   pname = "zfsbackup";
-  version = "unstable-2019-03-05";
-  rev = "78fea6e99f0a5a4c8513d3a3d1d45fb6750cfddf";
+  version = "unstable-2020-11-02";
```
Comment (xfix) [inline pkgs/tools/backup/zfsbackup/default.nix:5]:
```suggestion
  version = "unstable-2020-09-30";
```

May as well match a date of last commit.

**Review by sugar700 — APPROVED** [review]

**Comment by xfix** [comment]
This seems fine. I do run 2020-06-16 on my own server (https://gitlab.com/KonradBorowski/xfix.pw/-/blob/master/packages/zfsbackup-go/default.nix - not submitted to nixpkgs because https://github.com/NixOS/nixpkgs/pull/89338 was still waiting to be merged) and 2020-11-02 only changed documentation, so it should work as well.

File: pkgs/tools/backup/zfsbackup/deps.nix (hunk 1 above)
Comment (rajivr) [reply pkgs/tools/backup/zfsbackup/deps.nix:1]: This file was indeed generated using `vgo2nix`.

File: pkgs/tools/backup/zfsbackup/default.nix (hunk 2 above)
Comment (rajivr) [reply pkgs/tools/backup/zfsbackup/default.nix:5]: Sure! will push a new update with the suggested change.

**Comment by rajivr** [comment]
> This seems fine. I do run 2020-06-16 on my own server (https://gitlab.com/KonradBorowski/xfix.pw/-/blob/master/packages/zfsbackup-go/default.nix - not submitted to nixpkgs because #89338 was still waiting to be merged) and 2020-11-02 only changed documentation, so it should work as well.

Is there some reason #89338  was not merged?

**Review by sugar700 — APPROVED** [review]

**Pull request merged**